tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py

```diff
@@ -116,7 +116,8 @@ def test_list_recommendations_without_parameters(cluster_type):
     update_results_json = []
     update_results_json.append(result_json_arr[len(result_json_arr)-1])
 
-    expected_duration_in_hours = len(result_json_arr) * 15 / 60
+    # Expected duration in hours is 24h as for short term only 24h plus or minus 30s of data is considered to generate recommendations
+    expected_duration_in_hours = 24.0
```

Comment:
Instead of hard coding `24` everywhere, can we reference this back to `SHORT_TERM_DURATION_IN_HRS_MAX`